tests/compose/test-blacklist.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -xeuo pipefail
+
+dn=$(cd "$(dirname "$0")" && pwd)
+# shellcheck source=libcomposetest.sh
+. "${dn}/libcomposetest.sh"
+
+# Add a local rpm-md repo for recommends testing
+treefile_append "repos" '["test-repo"]'
+build_rpm foodep
+build_rpm foobar recommends foobar-rec requires foodep
+build_rpm foobar-rec
+
+echo gpgcheck=0 >> yumrepo.repo
+ln "$PWD/yumrepo.repo" config/yumrepo.repo
+# the top-level manifest doesn't have any packages, so just set it
```

Comment:
Minor: yeah, I know this is a cargo-culted comment from another test, but it's actually out-of-date. We operate on a flattened manifest. (I had changed `treefile_set` to `treefile_append` but forgot to nuke the comment :) ). Could just drop this and the other instance of it (or leave that as a follow-up).